### PR TITLE
add hover effect to client and role scopes tables

### PR DESCRIPTION
**Pull request by Jo-IE**
Hi @helfi92  I opened a new PR for #1403.

How do I fix the failing test?

### CODE REVIEW

File: ui/src/components/ClientScopesTable/index.jsx [hunk 1]
```diff
@@ -102,7 +104,7 @@ export default class ClientScopesTable extends Component {
   );
 
   renderRow = (scope, index) => {
-    const { searchTerm, classes, selectedScope } = this.props;
```
Comment (helfi92) [inline ui/src/components/ClientScopesTable/index.jsx]: Removing `classes` means we can remove `@withStyles` as well:

```jsx
@withStyles(theme => ({
  tableCell: {
    textDecoration: 'none',
  },
  listItemCell: {
    display: 'flex',
    justifyContent: 'space-between',
    width: '100%',
    padding: theme.spacing.unit,
    ...theme.mixins.hover,
  },
}))
```

File: ui/src/components/ClientScopesTable/index.jsx [hunk 2]
```diff
@@ -113,18 +115,18 @@ export default class ClientScopesTable extends Component {
...
                 selectedScope
                   ? `/auth/clients/${encodeURIComponent(node)}`
                   : `/auth/scopes/${encodeURIComponent(node)}`
               }>
-              <div className={classes.listItemCell}>
-                <Typography>{node}</Typography>
-                <LinkIcon size={16} />
-              </div>
-            </Link>
+              <ListItemText primary={<Typography>{node}</Typography>} />
```
Comment (helfi92) [inline ui/src/components/ClientScopesTable/index.jsx]:
```suggestion
              <ListItemText primary={node} />
```

Comment (helfi92) [reply ui/src/components/ClientScopesTable/index.jsx]: Then we could also stop importing the `Typography` component.

Comment (Jo-IE) [reply ui/src/components/ClientScopesTable/index.jsx]: ok thank you

**Pull request merged**